### ISSUE TITLE
Side nav disappears when in Keys section

### ISSUE THREAD

**Issue by lyubomir-popov** [canonical-web-and-design/maas-ui#394]
<img width="1424" alt="Screenshot 2019-11-04 at 17 46 20" src="https://user-images.githubusercontent.com/2741678/68204846-c545cb00-ffc0-11e9-8bf7-81b49ae55534.png">

**Comment by lilyvidenova** [comment]
This is as designed.

**Comment by Caleb-Ellis** [comment]
@lyubomir-popov I'm not sure I'm understanding this issue. What do you mean the side nav disappears? That should remain even when there is a form card.

**Comment by lyubomir-popov** [comment]
@Caleb-Ellis Turns out this was by design, so it shouldn't be part of this epic. Still, I think it is worth discussing later. I'm taking it out of the epic.

I think this will make more sense if I resort to a metaphor, so please bear with me.

Imagine you're in this hallway:

![danny-tricycle-the-shining-carpet-kubrick](https://user-images.githubusercontent.com/2741678/68467709-9cb31080-020e-11ea-9c9b-0fc6cceae374.png)

Walk into any of the rooms, close the door behind you, then reopen it, you end up in the same hallway.
Except for the first room on the left, next to the bin. If you walk through that door, then reopen it, you find yourself here:

![shining-206_resize](https://user-images.githubusercontent.com/2741678/68468036-2a8efb80-020f-11ea-8448-cbc7dfef0651.jpg)

This is what's happening in Settings at the moment:

Hallway 1: http://localhost:8400/MAAS/r/settings/users

![image](https://user-images.githubusercontent.com/2741678/68468337-b9037d00-020f-11ea-967c-27c6728bcf3f.png)

Halway 2: http://localhost:8400/MAAS/r/account/prefs

![image](https://user-images.githubusercontent.com/2741678/68468764-93c33e80-0210-11ea-9a29-46a33fd18dd1.png)

Any room but the first: http://localhost:8400/MAAS/r/settings/users/11/edit

![image](https://user-images.githubusercontent.com/2741678/68468306-aa1cca80-020f-11ea-94ca-d82ff7ca8db2.png)

First room:

![image](https://user-images.githubusercontent.com/2741678/68468258-95d8cd80-020f-11ea-91fc-f973e282565f.png)


---
The design rationale is, there are more options to edit for your own user, hence the context switch. In the terms of the above metaphor, that's like saying  "there's more furniture in that room, that's why the way into it is from one halway, but the only way out is onto a different one".

**Comment by lyubomir-popov** [comment]
One way around this is to use the top level user name (admin) link as a shortcut to "Me" in the Settings nav. Where Me is a child of users:

Users
   - Me
   - Others

**Comment by lilyvidenova** [comment]
This is an extremely niche issue.

I do not believe obscuring information (by moving it under Settings > Users > Me > Keys) about your user and any keys associated with that user will be useful or make the interactions more clear.

**Comment by huwshimi** [comment]
I wonder if a visual cue in the user list would help? Something that sets the expectation that editing your user will take you to a different section to the other edit forms. Maybe a different button or a tooltip or something.

**Comment by lyubomir-popov** [comment]
@lilyvidenova yes, it's a very low priority thing, that's why I've moved it out of the Design QA epic. Added a low priority label as well. 

If you don't mind I thing it should exist as an open issue rather than being closed as irrelevant, mainly because we need to keep track of little inconsistencies that spring up as a side effect of new design choices we're making. It is a litmus test of sorts of how scaleable those new choice are.

**Comment by lilyvidenova** [comment]
I don't mind it staying open. 

I just want to note that this is not a new design decision we have made, it has been true for a while. It was specifically done in response to a bug report on Launchpad stating it's very confusing that you can edit the same information in different formats from two distinct places.

**Comment by lyubomir-popov** [comment]
Oh, I see. Is [this](https://bugs.launchpad.net/maas/+bug/1754034) the LP bug?

**Comment by lilyvidenova** [comment]
Yes
